README.md

```diff
@@ -666,100 +795,161 @@ Specifies the addresses the server accepts connections to. Valid values:
+Passes a value through to the `package` resource when creating the server instance.
+
+Default value: `undef`.
 
 ##### `package_name`
 
-Specifies the name of the package to use for installing the server software. Default: OS dependent.
+Specifies the name of the package to use for installing the server software.
+
+Default value: OS dependent.
 
 ##### `pg_hba_conf_defaults`
 
-If false, disables the defaults supplied with the module for `pg_hba.conf`. This is useful if you disagree with the defaults and wish to override them yourself. Be sure that your changes of course align with the rest of the module, as some access is required to perform basic `psql` operations for example.
+If `false`, disables the defaults supplied with the module for `pg_hba.conf`. This is useful if you disagree with the defaults and wish to override them yourself. Be sure that your changes of course align with the rest of the module, as some access is required to perform basic `psql` operations for example.
 
 ##### `pg_hba_conf_path`
 
 Specifies the path to your `pg_hba.conf` file.
 
 ##### `pg_ident_conf_path`
 
-Specifies the path to your `pg_ident.conf` file. Default: "${confdir}/pg_ident.conf".
+Specifies the path to your `pg_ident.conf` file.
+
+Default value: '${confdir}/pg_ident.conf'.
 
 ##### `plperl_package_name`
 
-Sets the default package name for the PL/Perl extension. Default: OS dependent.
+Sets the default package name for the PL/Perl extension.
+
+Default value: OS dependent.
 
 ##### `plpython_package_name`
 
-Sets the default package name for the PL/Python extension. Default: OS dependent.
+Sets the default package name for the PL/Python extension.
+
+Default value: OS dependent.
 
 ##### `port`
 
-Specifies the port for the PostgreSQL server to listen on. **Note:** The same port number is used for all IP addresses the server listens on. Also, for Red Hat systems and early Debian systems, changing the port causes the server to come to a full stop before being able to make the change. Default: `5432`, meaning the Postgres server listens on TCP port 5432.
+Specifies the port for the PostgreSQL server to listen on. **Note:** The same port number is used for all IP addresses the server listens on. Also, for Red Hat systems and early Debian systems, changing the port causes the server to come to a full stop before being able to make the change. 
+
+Default value: 5432. Meaning the Postgres server listens on TCP port 5432.
 
 ##### `postgres_password`
 
-Sets the password for the `postgres` user to your specified value. Default: undef, meaning the superuser account in the Postgres database is a user called `postgres` and this account does not have a password.
+Sets the password for the `postgres` user to your specified value.
+
+Default value: `undef`, meaning the superuser account in the Postgres database is a user called `postgres` and this account does not have a password.
```

Comment:
Here, I would move the second half of this sentence up. So:

Sets the password for the `postgres` user to your specified value. By default, this setting uses the superuser account in the Postgres database, with a user called `postgres` and no password.

Default value: `undef`.